### PR TITLE
NOISSUE - Add policy addition endpoint

### PR DESCRIPTION
**Pull request by buraksekili**
Signed-off-by: Burak Sekili <buraksekili@gmail.com>

### What does this do?

This PR introduces a new endpoint on Auth package. This new endpoint allows an admin to create new policies.

Example:
```bash
curl -isSX POST http://localhost:<service_port>/policy -d '{"subjects":["<subject>"],"policies": ["<policy>"], "object": "<object>"}' -H "Authorization: <admin_auth_token>" -H 'Content-Type: application/json'
```

### Which issue(s) does this PR fix/relate to?
`-`

### List any changes that modify/break current functionality
`-`
### Have you included tests for your changes?
No, not yet

### Did you document any new/modified functionality?
`-`

### CODE REVIEW

File: auth/policies.go [hunk 1]
```diff
@@ -30,6 +30,10 @@ type Authz interface {
 	// error in case of failures.
 	AddPolicy(ctx context.Context, pr PolicyReq) error
 
+	// AddPolicyBulk adds new policies for given subjects. This method is
+	// only allowed to use as an admin.
+	AddPolicyBulk(ctx context.Context, token, object string, subjectIDs, relations []string) error
```
Comment (dborovcanin) [inline auth/policies.go]: Rename to `AddPolicies`.

File: auth/api/http/policies/transport.go [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+	"github.com/opentracing/opentracing-go"
+)
+
+var (
+	errUnsupportedContentType = errors.New("unsupported content type")
+)
+
+const (
+	contentType = "application/json"
+)
```
Comment (dborovcanin) [inline auth/api/http/policies/transport.go]:
```go
var errUnsupportedContentType = errors.New("unsupported content type")

const contentType = "application/json"
```

**Review by dborovcanin — CHANGES_REQUESTED** [review]
Please don't forget to add tests. You can do it in a separate PR.

File: auth/policies.go (hunk 1 above)
Comment (buraksekili) [reply auth/policies.go]: Updated

**Pull request merged**